src/driver/amdxdna/aie2_message.c

```diff
@@ -707,19 +707,26 @@ int aie2_execbuf(struct amdxdna_ctx *ctx, struct amdxdna_sched_job *job,
 
 	op = amdxdna_cmd_get_op(cmd_abo);
 	switch (op) {
-	case ERT_START_CU:
-		if (unlikely(payload_len > sizeof(req.ebuf.payload)))
-			XDNA_DBG(xdna, "Invalid ebuf payload len: %d", payload_len);
+	case ERT_START_CU: {
```

Comment:
In the Linux coding style, all the example of "case" in switch statement doesn't use curly bracket. The coding style doesn’t forbid the use of curly brackets. But to keep the coding style consistent in our code, please don't use curly bracket in this case.

We can use curly bracket on demand, for example, the rest of the cases here use curly bracket for local variable "payload".